packages/pagination/stories/2-Pagination.stories.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Pagination } from '@zendeskgarden/react-pagination';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Pagination/Pagination',
+  component: Pagination
+} as Meta;
+
+export const Default: Story = ({ currentPage, totalPages, pagePadding, pageGap }) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <Pagination
+          totalPages={totalPages}
+          pagePadding={pagePadding}
+          pageGap={pageGap}
+          currentPage={currentPage}
+          onChange={action('onChange')}
+        />
+      </Col>
+    </Row>
+  </Grid>
+);
+
+Default.args = {
+  currentPage: 1,
+  totalPages: 11,
+  pagePadding: 2,
+  pageGap: 2
+};
+
+Default.argTypes = {
+  currentPage: {
+    control: { type: 'number' }
```

Comment:
I went with a number control here because total pages can be changed to 1000 or 100000. It would allow a user to navigate to a specific page, e.g. 999 by typing it out. It also allows users to have negative numbers but that doesn't reflect in the UI and I thought that would be okay.